internal/logic/assembler/transfer_event_validator.go

```diff
@@ -310,9 +324,7 @@ func (t *TransferEventMatcher) erc721Matcher(transferEvents, gatewayEvents []eve
 				"gateway balance", info.GatewayBalance.String(),
 				"err info", info.Error,
 			)
-			slack.Notify(slack.MrkDwnGatewayTransferMessage(info))
-			return fmt.Errorf("erc721 mismatch for tokenAddress %s: gateway amount = %s, transfer amount = %s",
```

Comment:
why delete here

Reply:
because using a unified return handling function to handle ignored cases:
```
func (t *TransferEventMatcher) sendSlackAlert(info slack.GatewayTransferInfo) error {
	info.TokenIgnored = t.isTokenIgnored(info.Layer, info.TokenAddress)
	slack.Notify(slack.MrkDwnGatewayTransferMessage(info))
	if info.TokenIgnored {
		return nil
	}
	return fmt.Errorf("balance mismatch for token %s, token type = %s, transfer amount = %s, gateway amount = %s, info = %v",
		info.TokenAddress.Hex(), info.TokenType.String(), info.TransferBalance.String(), info.GatewayBalance.String(), info)
}
```

if this token is ignored, then only send an alert, with the "ignored: true" flag.
if this token is not ignored, will return an error to halt chain-monitor.